csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -241,31 +241,39 @@ module Public {
     }
 
     /**
-     * Holds if the summary is auto generated and not manually generated.
+     * Holds if the summaries that apply to `this` are auto generated and not manually generated.
```

Comment:
Minor: Does "manually generated" make sense? `manual` models are not generated, are they.

Reply:
Ah yes, I will change the wording to "manually created".